parsons/action_network/action_network.py

```diff
@@ -200,8 +204,25 @@ def add_person(self, email_address=None, given_name=None, family_name=None, tags
         logger.info(f"Entry {person_id} successfully added to people.")
         return response
 
+    def add_person(self, email_address=None, given_name=None, family_name=None, tags=None,
+                   languages_spoken=None, postal_addresses=None, mobile_number=None,
+                   mobile_status='subscribed', **kwargs):
+        """
+        Creates a person in the database. WARNING: this endpoint has been deprecated in favor of
+        upsert_person.
+        """
+        logger.warning("Method 'add_person' has been deprecated. Please use 'upsert_person'.")
+        # Pass inputs to preferred method:
+        self.upsert_person(email_address=email_address, given_name=given_name,
+                           family_name=family_name, languages_spoken=languages_spoken,
+                           postal_addresses=postal_addresses, mobile_number=mobile_number,
+                           mobile_status=mobile_status, **kwargs)
+
     def update_person(self, entry_id, **kwargs):
         """
+        Updates a person's data in Action Network. WARNING: this endpoint has been deprecated in
+        favor of upsert_person and may not work properly.
+
```

Comment:
Now that we know that the person-specific endpoint _does_ work with `PUT` requests, do we still think that we want to deprecate this? As an alternative, we could just remove the tagging functionality. For instance, NGPVAN has both upsert and update methods. On the other hand, maybe keeping things simpler is a better approach?

Reply:
Do you think the person-specific endpoint is easier to use? If so, we should probably keep it.

Reply:
I have no reason to believe it is, but I'm afraid I also don't have any reason to believe it _isn't_. The only thing I know for sure is that the method we're changing to upsert (i.e. the one that uses the helper endpoint) is _definitely_ used, and I have no notion of whether `update_person()` is used at all. Honestly, I think it matters very little, so I was hoping you'd have a structural, conventional, or even esthetic reason to help make the call.

Reply:
Hmm, yeah. I think conventionally/aesthetically you minimize duplicate calls, but otoh we don't want to break things for people.